actions.py

```diff
@@ -169,6 +215,11 @@ def calculate_emissions(self, tracker, unit="short_tons"):
             raise ValueError("Destination IATA code unknown.")
         if not flight_class:
             raise ValueError("Flight class unknown.")
```

Comment:
Don't really need this conditional given the one afterwards (which also has a nicer error message)

Reply:
True 🙂